src/views/Submit.vue

```diff
@@ -261,11 +283,23 @@ export default {
 			// Using button-click event to not bypass validity-checks and use our specified behaviour
 			this.$refs.submitButton.click()
 		},
+		addFormFieldToLocalStorage(question) {
+
+			this.formValuesForLocalStorage[question.id] = { id: question.id, value: this.answers[question.id], type: answerTypes[question.type].component.name }
```

Comment:
Unneeded extra new line:

```suggestion
		addFormFieldToLocalStorage(question) {
			this.formValuesForLocalStorage[question.id] = { id: question.id, value: this.answers[question.id], type: answerTypes[question.type].component.name }
```